app/Http/Controllers/ChopsController.php

```diff
@@ -16,107 +17,109 @@
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Support\Facades\Redirect;
+use Illuminate\Support\Facades\Input as Input;
+use Validator;
+use Response;
 use Cloudder;
 
+\Cloudinary::config(array(
+    "cloud_name" => "chopbox"
+));
```

Comment:
Pls remove it from there...that's dirty! :-1:  you can have a function that just retrieves it from the environment variable or from the Cloudinary config file